vercel.json

```diff
@@ -0,0 +1,7 @@
+{
+  "build": {
+    "env": {
+      "NODEJS_AWS_HANDLER_NAME": "handler"
```

Comment:
This is needed so Vercel treats the serverless function as an AWS Lambda function. See https://vercel.com/docs/runtimes#advanced-usage/advanced-node-js-usage/aws-lambda-api and the docstring in `api/index.js`.